pandas/core/strings.py

```diff
@@ -1385,17 +1385,40 @@ def str_rsplit(arr, pat=None, n=None):
 
 def str_slice(arr, start=None, stop=None, step=None):
     """
-    Slice substrings from each element in the Series/Index
+    Slice substrings from each element in the Series or Index
 
     Parameters
     ----------
     start : int or None
+        Start position for slice operation
     stop : int or None
+        Stop position for slice operation
     step : int or None
+        Step size for slice operation
 
     Returns
     -------
-    sliced : Series/Index of objects
+    Series or Index of object
+        Series or Index containing sliced substring from original string object
+
+    Examples
+    --------
+    >>> s = pd.Series(["panda", "fox"])
+    >>> s
+    0    panda
+    1      fox
+
+    >>> s.str.slice(start=2)
+    0    nda
+    1      x
+
+    >>> s.str.slice(stop=2)
+    0    pa
+    1    fo
+
+    >>> s.str.slice(step=2)
+    0    pna
+    1     fx
```

Comment:
May be it's worth showing that `s.str[2:-2]` (or whatever slice) is equivalent to `s.str.slice(start=2, stop=-2)`.